packages/plugins/cloud/admin/src/components/CloudDeploy.tsx

```diff
@@ -0,0 +1,156 @@
+/*
+ *
+ * Cloud component
+ *
+ */
+
+import {
+  Box,
+  Flex,
+  Typography,
+  Link,
+  LinkButton,
+  Tabs,
+  Badge,
+  BaseLink,
+} from '@strapi/design-system';
+import { ExternalLink, CloudUpload, Code } from '@strapi/icons';
+import { useIntl } from 'react-intl';
+
+import { CLIDeployTabs } from '../components/CLIDeployTabs';
+import { getTrad } from '../utils/getTrad';
+
+const CloudDeploy = () => {
+  const { formatMessage } = useIntl();
+
+  const documentationLink = formatMessage({
+    id: getTrad('Homepage.deploy.documentation'),
+    defaultMessage: 'Having trouble? Check our documentation',
+  });
+
+  return (
+    <Box paddingBottom={5}>
+      <Flex direction={'column'}>
```

Comment:
We try to use strings where possible but it's not a big deal 

```suggestion
      <Flex direction="column">
```

Reply:
Ok I can fix! Anywhere else aside from the column/row?

Reply:
anywhere you are passing a string as a value to a prop. I think it's a lot of places 😬. It's really a style preference thing so up to you.

Reply:
Updated